RELEASE_NOTES.md

```diff
@@ -74,6 +74,49 @@ bundle exec rake decidim:upgrade:moderation:fix_blocked_user_panel
 
 You can read more about this change on PR [\#10521](https://github.com/decidim/decidim/pull/10521).
 
+### 3.3 Change Webpacker to Shakapacker
+
+Since the Rails team has retired the Webpacker in favour or importmap-rails or js-bundling, we got ouserlves in a situation where performance improvements could not be performed.
+In order to continue having support for Webpacker like syntax, we have switched to Shakapacker.
+
+In order to perform the update, you will need to make sure that you **do not have webpacker in your Gemfile**.
+If you have it, please remove it, adn allow Decidim to handle the webpacker / shackapacker dependency.
+
+In order to perform the migration to shakapacker, please backup the following files, to make sure that you save any customizations you may have done to webpacker:
+
+```console
+config/webpacker.yml
+config/webpack/*
+package.json
+postcss.config.js
+```
+
+After all the backups and changes mentioned above have been completed, follow the default upgrade steps, as mentioned above in the document.
+Then run the below command, and replace all the configuration with the one that Decidim is providing by default:
+
+```console
+bundle exec rake decidim:webpacker:install
+```
+
+This will make the necessary changes in the `config/webpacker.yml`, but also in the `config/webpack/` folder.
+
+#### Note for development
+
+Compiling the JS and CSS at the runtime is a very slow process. Additionally, compiling JS and CSS in the test suite will cause your pipeline to timeout.
+To avoid that, you could export the following env variable:
```

Comment:
I think this does not apply anymore or this comment has to be inversed since we set the default value to `false`.

We should also advice here to use `./bin/dev` to run the development server + webpacker dev server.